### PR TITLE
[Bug]Support get all rowset meta info in memory from tablet meta url

### PR DESCRIPTION
**Pull request by caiconghui**
This PR is to fix bug that we cannot get the newest tablet meta info from tablet meta url.

### CODE REVIEW

**Comment by caiconghui** [comment]
for #4061

File: be/src/http/action/meta_action.cpp [hunk 1]
```diff
@@ -60,13 +60,11 @@ Status MetaAction::_handle_header(HttpRequest* req, std::string* json_meta) {
         LOG(WARNING) << "no tablet for tablet_id:" << tablet_id << " schema hash:" << schema_hash;
         return Status::InternalError("no tablet exist");
     }
-    OLAPStatus s =
-            TabletMetaManager::get_json_meta(tablet->data_dir(), tablet_id, schema_hash, json_meta);
```
Comment (morningman) [inline be/src/http/action/meta_action.cpp:64]: Why are some rowset meta not in rocksdb but only in memory?
Is this part of rowset meta not effective?

Comment (caiconghui) [reply be/src/http/action/meta_action.cpp:64]: this problem is introduced by #1946 , tablet will do meta checkpoint periodly

**Review by chaoyli — APPROVED** [review]

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**